S4TF_Tutorial_4.ipynb

```diff
@@ -37,7 +37,7 @@
       "metadata": {
```

Comment:
For this notebook, it looks like many of the comments I made on notebook 3 can be brought across, too.